src/style/colors.ts

```diff
@@ -49,10 +49,10 @@ export class Colors {
    * Load light and dark schemes based on generated design tokens
    * @param color - palette color
    */
-  loadDesignTokens(color: string) {
+  loadDesignTokens({primaryColor}: {primaryColor: string}) {
```

Comment:
Why not make the type a defined type, like `export type TokensOptions: {primaryColor: string}`, and have a parameter like `loadDesignTokens(options: TokensOptions)`? Then you can use `options?.primaryColor` etc.

Reply:
Done